modules/ROOT/pages/proxy-deploy-runtime-fabric.adoc

```diff
@@ -47,7 +47,7 @@ After deployment, you can manage the deployed API proxy from Runtime Manager.
 To enable the proxy to support HTTPs requests:
 
 . Download the API gateway and deploy from API Manager using the instructions in xref:api-manager::download-proxy-task.adoc[Downloading an API Proxy].
-. Open the gateway in Anypoint Studio and insert a keystore (SSL cert) using the instructions in xref::mule-runtime::tls-configuration#use-anypoint-studio-to-configure-tls[Use Anypoint Studio to Configure TLS].
+. Open the gateway in Anypoint Studio and insert a keystore (SSL cert) using the instructions in xref:mule-runtime::tls-configuration#use-anypoint-studio-to-configure-tls[Use Anypoint Studio to Configure TLS].
```

Comment:
I think you need a .adoc extension somewhere in here.

Reply:
Surprisingly no - this works without the .adoc extension.